src/pages/IstioRuleDetails/IstioRuleInfo/IstioRuleInfoDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import PfInfoCard from '../../../components/Pf/PfInfoCard';
+import { Col, Row } from 'patternfly-react';
+
+interface IstioInfoDescriptionProps {
```

Comment:
Looking at `RuleItem` and parent component usage, `name` and `match` should not be optional, or did I miss something?

Reply:
Not sure if I understand the question.
This one is a mapping that comes from the istio rule details
```
      let data = response['data'];
      this.setState({
        name: data.name,
        match: data.match,
        actions: data.actions
      });
```
Meanwhile RuleItem is used in the rules list page to map the list of rules.

Reply:
Yeah what I mean is why are they declared optional `name?: string` instead of just `name: string` ?

Reply:
Ah, true, name is always there, but match can be empty, I'll fix the name one.
Thx !